src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java

```diff
@@ -21,6 +25,9 @@
     private final BbFuelConfiguration config;
     private static final String SERVICE_VERSION = "v2";
     private static final String ENDPOINT_USERS = "/users";
+    private static final String ENDPOINT_IDENTITIES = ENDPOINT_USERS + "/identities";
+
+    private final UserPresentationRestClient userPresentationRestClient;
```

Comment:
doesn't fell right, making one Rest Client dependant on the other one

Reply:
yea that does seem odd. Also should this remove all references to the legacy triangular architecture ?

Reply:
Agree, but that should probably be part of a bigger effort to "rename" BB-Fuel clients that I am not sure should be the scope of this ticket